Jenkinsfile

```diff
@@ -142,6 +142,12 @@ stage('Configure') {
 							condition: TestCondition.AFTER_MERGE),
 					new JdkBuildEnvironment(version: '16', buildJdkTool: 'OpenJDK 16 Latest',
 							condition: TestCondition.AFTER_MERGE)
+			],
+			wildflyTck: [
+					new WildFlyTckBuildEnvironment(javaVersion: '8', buildJdkTool: 'OracleJDK8 Latest',
+							condition: TestCondition.AFTER_MERGE),
+					new WildFlyTckBuildEnvironment(javaVersion: '11', buildJdkTool: 'OpenJDK 11 Latest',
+							condition: TestCondition.AFTER_MERGE)
```

Comment:
OK, apparently, you partially answered that. What about the sigtest and jqassistant?

I suppose `TestCondition.AFTER_MERGE` means this will be only built when a commit is merged?

Reply:
> What about the sigtest and jqassistant?

Personally on Search, I just run JQAssistant as part of the default build (JDK8). Maybe we could do that?

About sigtest, no idea, I don't use it. But sure, we could include it somewhere in the Jenkinsfile.

> I suppose TestCondition.AFTER_MERGE means this will be only built when a commit is merged?

Yes. There are comments further down, on the enum, to explain the exact meaning. And I think I explained the general idea in the PR description.

Reply:
As per our discussion on Zulip, I added sigtest and JQAssistant to the default build. It will run for every build, including PRs. We can talk about moving this to separate stages in another PR.